lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,32 +1,32 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-const Tab = ({ tab, selectTabHandler}) => {
-  // Make sure you include PropTypes on your props.
-  Tab.propTypes = {
-    tab: PropTypes.string,
-    selectTabHandler: PropTypes.func
-  };
+const Tab = props => {
 
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
 
-  
   return (
-    <div className="tab active-tab"
-      //if tab === selectedtab { return className=" active-tab"}
-     // else return  className="tab"
+    <div 
+    className={props.selectedTab === props.tab ? 'tab active-tab' : 'tab'}
+      //if (tab === selectedtab { return className=" active-tab"}
+     // else return  className="tab");
 
       onClick={() => {
         /* Replace this dummy click handler function with your selectTabHandler function from props 
          you'll need to pass the `tab` in as an argument to this handler. */
-        selectTabHandler(tab);
+        props.selectTabHandler(props.tab);
```

Comment:
Good Code here Antonio. Could you explain the use of props here and what benefit they give you?